nightingale_navigation/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ catkin_package(
 ## in contrast to setup.py, you can choose the destination
 catkin_install_python(PROGRAMS
   src/room_runner.py
+  src/test_do_not_merge.py
```

Comment:
navigation related changes will be removed from this PR and merged in a different PR